src/ray/raylet/task_spec.h

```diff
@@ -155,6 +155,13 @@ class TaskSpecification {
   flatbuffers::Offset<flatbuffers::String> ToFlatbuffer(
       flatbuffers::FlatBufferBuilder &fbb) const;
 
+  std::string ToFlatbuffer() const {
```

Comment:
`SerializeAsFlatbufferString`? Or `SerializeAsString` or something like that?

Reply:
I'm doing SerializeAsString to be consistent with SerializeTaskAsString